app-admin/abrt/abrt-2.10.9.ebuild

```diff
@@ -0,0 +1,115 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6} )
+
+inherit autotools xdg-utils python-single-r1 systemd user
+
+DESCRIPTION="Automatic bug detection and reporting tool"
+HOMEPAGE="https://github.com/abrt/abrt/wiki/ABRT-Project"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 x86"
+IUSE="debug python selinux systemd"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+COMMON_DEPEND="${PYTHON_DEPS}
+	>=dev-libs/btparser-0.18
+	>=dev-libs/glib-2.43:2
+	>=dev-libs/libreport-2.9.5[python]
+	dev-libs/libxml2
+	dev-libs/nss
+	>=gnome-base/gsettings-desktop-schemas-3.15.1
+	python? ( ${PYTHON_DEPS} )
```

Comment:
Python support is actually mandatory (and only py3 btw) as this is needed for cli-ng abrt script.